packages/cli/src/WebhookHelpers.ts

```diff
@@ -590,8 +590,7 @@ export async function executeWebhook(
 		}
 
 		// Start now to run the workflow
-		const workflowRunner = new WorkflowRunner();
```

Comment:
Interesting, intrigued to imagine why we wanted to instantiate this every time versus using it as a singleton. Not a major thing, but certainly a performance improvement.